website/src/models/emailUnsubscription.js

```diff
@@ -1,14 +1,20 @@
-var mongoose = require("mongoose");
-var shared = require('../../../common');
+import mongoose from 'mongoose';
+import common from '../../../common';
+import validator from 'validator';
 
 // A collection used to store mailing list unsubscription for non registered email addresses
-var EmailUnsubscriptionSchema = new mongoose.Schema({
+export let schema = new mongoose.Schema({
   _id: {
     type: String,
-    'default': shared.uuid
+    default: common.uuid,
```

Comment:
Should we be more careful about reserved words? That's the reason we had it quoted before.